ci-operator/step-registry/openshift-qe/installer/azure/openshift-qe-installer-azure-workflow.yaml

```diff
@@ -0,0 +1,13 @@
+workflow:
+  as: openshift-qe-installer-azure
+  steps:
+    env: 
+      COMPUTE_NODE_TYPE: Standard_D8s_v3
```

Comment:
Let's move the node sizing to the config layer so that anyone using the workflow doesn't have to use this size nodes